src/gui/owncloudgui.cpp

```diff
@@ -544,12 +547,16 @@ void ownCloudGui::slotShowGuiMessage(const QString &title, const QString &messag
 
 void ownCloudGui::slotShowSettings()
 {
-    if (_settingsDialog.isNull()) {
-        _settingsDialog = new SettingsDialog(this);
-        _settingsDialog->setAttribute(Qt::WA_DeleteOnClose, true);
-        _settingsDialog->show();
+    if (!AccountManager::instance()->accounts().isEmpty()) {
+        if (_settingsDialog.isNull()) {
+            _settingsDialog = new SettingsDialog(this);
+            _settingsDialog->setAttribute(Qt::WA_DeleteOnClose, true);
+            _settingsDialog->show();
+        }
+        raiseDialog(_settingsDialog.data());
+    } else {
+        slotNewAccountWizard();
     }
-    raiseDialog(_settingsDialog.data());
 }
```

Comment:
Tested on Windows and works fine except when no accounts are configured, then the Settings option only brings up the account wizard.

Reply:
Admittedly it's a side-effect this fix wasn't supposed to tackle... but to be fair, isn't it what we'd want nowadays? Because you can't add accounts from the settings dialog anymore it's triggered from the main dialog. Ah but then you wouldn't be able to set a proxy before creating an account... that could be a problem I guess.

Maybe that chunk should be reverted then?

Reply:
Reverted.